bal_addresses/errors.py

```diff
@@ -0,0 +1,8 @@
+
+class Errors():
```

Comment:
Not very pythonic. You just declare errors outside Errors class
```suggestion
    class MultipleMatchesError(Exception):
        pass

    class NoResultError(Exception):
        pass
```

Reply:
The thing is that the errors often need to be caught as part of the usage pattern, and it's kind of a pain to have to specifiy each error/track them all at the top level/not in a class.

Is there a better way to handle this.

https://github.com/BalancerMaxis/multisig-ops/blob/989d99d459ef2bbc687d8150e2e29d61f8a717d4/action-scripts/gen_add_permissions_payload.py#L69

Should the error be inside the AddrBook and BalPermissions classeS?  Then have the same error twice.

Reply:
I don't get what you mean. Why not just:
```python
from from bal_addresses.errors import NoResultError
...
except NoResultError as err:
```
???